autogpt_platform/frontend/src/components/edit/control/BlocksControl.tsx

```diff
@@ -219,7 +219,7 @@ export const BlocksControl: React.FC<BlocksControlProps> = ({
           </CardHeader>
           <CardContent className="overflow-scroll border-t border-t-gray-200 p-0 dark:border-t-slate-700">
             <ScrollArea
-              className="h-[60vh] w-fit"
+              className="h-[60vh] w-full"
```

Comment:
I see that you were adding this here: https://github.com/Significant-Gravitas/AutoGPT/pull/8874/files#diff-cab128493f8312915fb4439e83ea5ba6a0f07bb666ebe17ee95499eedd77a1e0R222

@Swiftyos let me know if this is fine to revert.

It makes the entry looks like this on SDM block entry:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/ad305f5c-5a3f-4c74-b6c6-cd92fe0368a7" />